weave-js/src/components/PagePanelComponents/Home/Browse3/pages/CallsPage/callsTableQuery.ts

```diff
@@ -32,10 +32,12 @@ export const useCallsForQuery = (
   project: string,
   filter: WFHighLevelCallFilter,
   gridFilter: GridFilterModel,
-  gridSort: GridSortModel,
-  gridPage: GridPaginationModel,
-  expandedColumns: Set<string>,
-  columns?: string[]
+  gridSort?: GridSortModel,
+  gridPage?: GridPaginationModel,
+  expandedColumns?: Set<string>,
+  columns?: string[],
+  offset?: number,
```

Comment:
nit: i don't prefer having parameters that override other parameters. I think you can just pass in a GridPaginationModel confirming payload. Where pageSize = limit and page = offset // limit

```
export interface GridPaginationModel {
    /**
     * Set the number of rows in one page.
     * If some of the rows have children (for instance in the tree data), this number represents the amount of top level rows wanted on each page.
     * @default 100
     */
    pageSize: number;
    /**
     * The zero-based index of the current page.
     * @default 0
     */
    page: number;
}
```